fs.go

```diff
@@ -178,6 +178,8 @@ type File interface {
 	Unlock() error
 	// Truncate the file.
 	Truncate(size int64) error
+	// Commit the current contents of the file to stable storage.
+	Sync() error
```

Comment:
Instead of expanding the existing interface, let's create a new interface for this:

type Syncer interface {
    Sync() error
}